docs/release-notes.md

```diff
@@ -23,6 +23,10 @@ UNRELEASED
 
 ##### Breaking changes since 24
 
+* The `frame` parameter of the `MatchResult` returned from `stbt.match`,
+  `stbt.detect_match` and `stbt.wait_for_match` is now read-only. Use
+  `numpy.copy(match_result.frame)` if you need a writable copy.
```

Comment:
I used `numpy.copy` because it has a different default (order='K' which means keep the same order) than `ndarray.copy` (order='C' which means row-major order). But order='C' is the default for new arrays, and all our frames are in that order anyway, so I've changed the release note to suggest `frame.copy()`.